pkg/services/cloudmigration/cloudmigrationimpl/xorm_store.go

```diff
@@ -259,11 +271,14 @@ func (ss *sqlStore) GetSnapshotByUID(ctx context.Context, sessionUid, uid string
 }
 
 // GetSnapshotList returns snapshots without resources included. Use GetSnapshotByUID to get individual snapshot results.
+// passing a limit of -1 will return all the elements regardless of the page
```

Comment:
I think we should add a constant in here so that we can pass in `cloudmigrationimpl.AllSnapshots` instead